pkg/graph/schema.graphql

```diff
@@ -7,6 +7,14 @@ type UserError {
 An accessibility request represents
 """
 type AccessibilityRequest {
+  id: UUID!
+  system: System!
```

Comment:
Moving from non-nullable to nullable is a real concern, but since we control the API and its only consumer, I think erring on the side of making things more strict upfront is worth it to have fewer cases to handle in the code.

Reply:
We need the connection to a system to display the appropriate data to the user, so I am ok with keeping this non-nullable.